open-api.yaml

```diff
@@ -0,0 +1,269 @@
+openapi: 3.0.3
+info:
+  title: ATLAS feeding_data
+  description: Send planned rations and get actual feeding reports into the system.
+  version: '0.0.2'
+servers:
+  - url: http://localhost:8080
+    description: Inferred Url
+security: 
+  - "bearerAuth": []
+paths:
+  /feedingreport:
+    get:
+      summary: feedingReport
+      parameters:
+        - name: start_date
+          in: query
+          description: Start date from when to fetch data, ISO8601 date
+          required: true
+          schema:
+            type: string
+            format: date
+        - name: end_date
+          in: query
+          description: End date from when to fetch data, ISO8601 date
+          required: true
+          schema:
+            type: string
+            format: date
+        - name: ration_ref
+          in: query
+          description: ration_ref represents identification for connecting the rations between systems

```

Comment:
ration_ref is an optional identifier to connect rations between different systems